src/applications/vaos/newAppointmentFlow.js

```diff
@@ -217,10 +217,6 @@ export default {
     async next(state, dispatch) {
       const eligibilityStatus = getEligibilityStatus(state);
 
-      if (!eligibilityStatus.direct && !eligibilityStatus.request) {
```

Comment:
This is only going to fire when there's an error. I'm not really sure where to put this event (even though I think I suggested it), so I just took it out for now.